### ISSUE TITLE
Export to schematic not working on kicad-7

### ISSUE THREAD

**Issue by ketan** [Bouni/kicad-jlcpcb-tools#286]
Export to schematic does not export LCSC details to schematic symbols. Incidentally I have managed to fix this, and will submit a PR tomorrow after testing this out locally a bit.

**To Reproduce**

1. Click on "export to schematic" after assigning a few LCSC numbers

**Expected behavior**

2. Schematic should contain LCSC part numbers on parts that I had selected in the JLCPCB tools plugin window

# KiCad Version

```
Application: KiCad PCB Editor x86_64 on x86_64

Version: (7.0.0-0), release build

Libraries:
	wxWidgets 3.2.1
	FreeType 2.12.1
	HarfBuzz 5.3.1
	FontConfig 2.14.0
	libcurl/7.86.0 (SecureTransport) LibreSSL/3.3.6 zlib/1.2.11 nghttp2/1.47.0

Platform: macOS Ventura Version 13.2 (Build 22D49), 64 bit, Little endian, wxMac

Build Info:
	Date: Feb 12 2023 11:39:44
	wxWidgets: 3.2.1 (wchar_t,wx containers)
	Boost: 1.80.0
	OCC: 7.6.3
	Curl: 7.77.0
	ngspice: 38
	Compiler: Clang 13.0.0 with C++ ABI 1002

Build settings:
	KICAD_SPICE=ON

```

For additional details. I added some logging information to `schematicexport.py`, and it appears that the regex `'\\(property\\s\\"(.*)\\"\\s\\"(.*)\\"\\s\\(id\\s(\\d+)\\)\\s\\(at\\s(-?\\d+(?:.\\d+)?\\s-?\\d+(?:.\\d+)?)\\s\\d+\\)'` to match the property line is not working as expected with the following text snippet from the schematic:

```
  (symbol (lib_id "Device:C_Small") (at 81.915 142.24 0) (unit 1)
    (in_bom yes) (on_board yes) (dnp no) (fields_autoplaced)
    (uuid edb3e5a0-e587-400b-8286-20e4b6fe62d6)
    (property "Reference" "C1" (at 84.2391 141.6026 0)
      (effects (font (size 1.27 1.27)) (justify left))
    )
    (property "Value" "100nF" (at 84.2391 143.5236 0)
      (effects (font (size 1.27 1.27)) (justify left))
    )
    (property "Footprint" "PCM_Capacitor_SMD_Handsoldering_AKL:C_0402_1005Metric" (at 81.915 142.24 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (property "Datasheet" "~" (at 81.915 142.24 0)
      (effects (font (size 1.27 1.27)) hide)
    )
    (pin "1" (uuid 01a41ea5-8e4c-4da0-9b82-d9916bfb385e))
    (pin "2" (uuid 4c180ac8-1539-42e9-984c-be7973f69199))
    (instances
      (project "Vario"
        (path "/e4ff166b-55ce-4271-89b3-b6dbb637a94d/ea947222-e8f4-47cc-877a-832cc47c42e9"
          (reference "C1") (unit 1)
        )
      )
    )
  )
```

**Comment by Bouni** [comment]
To be honest I've never tested this feature. It was created by a contributor. 
Would be awesome to get a PR that fixes the problem 😄

**Comment by computergeek1507** [comment]
I have a fix for this, the schematic format changed in 7.0. I will try to commit it soon.

**Comment by Bouni** [comment]
Fixed in #298
